### ISSUE TITLE
Ckan and Cygnus Compatibility Issue

### ISSUE THREAD

**Issue by Gaurav-sen** [telefonicaid/fiware-cygnus#1987]
Hello,
I hope everyone is doing well.

I am working on a deployment  which consists of ckan and cygnus components (including Postgres, Redis and Solr). This is deployment on AWS cloud over EKS cluster. 
I use ckan 2.8 version , whenever i deployed Cygnus latest version or its prior versions except 1.15.0, my pod is got stuck in crashLoopBackOff state after 5-10 mins of deployment with below error message.

**All the channels are empty
Stopping channels
Stopping ckan-channel (lyfecycle state=START)
Stopping sinks**

I am not able to find the root cause of this problem.

**Note : Cygnus 1.15,0 is working fine without any issue**

Kindly tell me, is it the version compatibility issue between Ckan 2.8 and Cygnus versions (except 1.15.0) or something else.

Thanks in advance.

Best Regards
Gaurav Sen

**Comment by AlvaroVega** [comment]
Could you provide us any log about cygnus docker before the crash ?

**Comment by Gaurav-sen** [comment]
![Logs_1](https://user-images.githubusercontent.com/20534368/102020277-23d1a100-3d9e-11eb-85e2-77956996afbf.PNG)
![Logs_2](https://user-images.githubusercontent.com/20534368/102020278-2502ce00-3d9e-11eb-95c2-bf9224e21d27.PNG)

See the attached screenshots, and let me know if more information is required.

**Comment by fgalan** [comment]
>  I use ckan 2.8 version , whenever i deployed Cygnus latest version or its prior versions except 1.15.0, my pod is got stuck in crashLoopBackOff state after 5-10 mins of deployment with below error message.

Not sure if I understand this correctly...

Do you have tested with latest Cygnus version available (2.6.0)? Take into account that 1.15.0 is a pretty old version (more than a year), its usage is not recommended.

In addition,

**Comment by Gaurav-sen** [comment]
> Do you have tested with latest Cygnus version available (2.6.0)?
Yes, i have tested with latest cygnus version, but getting the same error.
